admin_guide/ipsec.adoc

```diff
@@ -58,14 +58,14 @@ made aware of the change by removing the SDN interface and restarting the
 +
 ----
 # systemctl stop atomic-openshift-node
-# ovs-vsctl del-br br0
+# oc exec ovs-vsctl del-br br0
 # systemctl start atomic-openshift-node
 ----
 +
 .. Remove the SDN interface
 +
 ----
-# ovs-vsctl del-br br0
+# oc exec ovs-vsctl del-br br0
```

Comment:
I have this there now. Does that work?

`# oc exec <SDN/OVS podname> -- ovs-vsctl del-br br0`

Reply:
Neal, this seems okay but i suggest to just include ovs pod here to avoid confusion. As we really need only need this command on OVS and corresponding sdn pod will automatically get a restart. So lets do **# oc exec <ovs_pod_name> -- ovs-vsctl del-br br0**. Also, Put a note below saying "this needs to be executed on all ovs pods one by one". Sounds okay?

Reply:
rest is lgtm to me

Reply:
Reason is ,this only either needs to be executed on all sdn pods or all ovs pods. So lets stick to ovs as ovs-vsctl is actually an ovs commands so make sense in that pods.

Reply:
Ok, made that change.